### DISCUSSION TITLE
Motion detection question

### DISCUSSION THREAD

**Discussion by oneseventhree** [fuatakgun/eufy_security#565] [Q&A]
Hi All,

How quickly does the motion detection work?

I currently hve a reolink camera and hate it because the motion detection takes 20 seconds or never to trigger that there was motion. I want to use my camera to turn on my light when I go to the backyard.

I am currently using a motion sensor from Aqara because the reolink is so bad at it.

Does thie integration have fast motion detecition? is there also person detection if I go for the 2k outdoor cam (T8441C21) ?

Thanks in advance!

**Comment by fuatakgun** [comment]
couple of seconds (2 to 3)
